### ISSUE TITLE
My characters auto jailed

### ISSUE THREAD

**Issue by zenoooop** [LegionDark/Issues#1739]
Date & Time:
12/19/2017 9:00 am

FFXI Client Version (use /ver):
 newest version

Server's Expected Client Version matches, yes/no? (use !ecv):

Character Name:
Zen

Nation:
sandy

Job(level)/Sub Job(level):
99 sam, blu,war

NPC or Monster or item Name:

Zone name:

Coordinates (use !where):

ffxiah.com link (for items issues only):

Multi-boxing? (multiple clients on same connection):
YES

Steps To Reproduce / any other info:

Was testing debuff addon. did /cancel 0, /cancel 1, then got sent to jail. Read here that we could cancel weakness debuff with the addon. Didn't realize that. Addon says we couldn't cancel debuff. Just need help getting out. Thanks

![zen_2017 12 19_153501](https://user-images.githubusercontent.com/34695797/34184100-d9cc36aa-e4d2-11e7-9fcc-af01579c2c54.png)

**Comment by EpicTaru** [comment]
This happens from using tools or addons that try to strip off status effects that you normally cannot.

Each time that happens the code will re-jail. Even if you are already in jail. Please disable any such tools and remove them and any windower or ashita scripts or macros that would attempt to automatically remove such effects (like weakness) so that you don't auto-jail yourself again, and I will release you at my earliest convenience. And hopefully I never see you jail yourself again.

tl;dr
A GM will be with you when one gets the chance to

**Comment by EpicTaru** [comment]
Also, WHERE HERE at LegionDark did you read that it is ok to cancel debuffs?

**Comment by zenoooop** [comment]
Nah, I meant that I read it here (github) that it was possible to cancel weakness which was not consist with atom0s's description of the addon. He specifies, "This addon cannot remove debuffs on your character! You can only cancel buffs that can normally be clicked off."  Of course, I realize it would be detrimental to the server if players can just forgo weakness.

**Comment by EpicTaru** [comment]
@zenoooop so have you disabled the relevant addon/plugin?

**Comment by zenoooop** [comment]
@EpicTaru yea, deleted that addon

**Comment by EpicTaru** [comment]
Released 12/20.
